src/index.html

```diff
@@ -9,6 +9,172 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Human rights</h1>
+<h1 class="page-tittle">
+    Universal Declaration of Human Rights
+</h1>
+<article>
+<h2 class="article-tittle">
```

Comment:
Follow the indentation rules. Add an additional space of 2 spaces for nested tags. 4 - is too much.